server/src/main/java/io/druid/segment/realtime/appenderator/Appenderator.java

```diff
@@ -53,6 +53,14 @@
   Object startJob();
 
   /**
+   * Returns the size of data currently stored in local storage.  The persistedBytes is changed when
+   * {@link #persistAll(Committer)} and {@link #drop(SegmentIdentifier)} is called.
```

Comment:
I tried to trace the actual byte size of segments stored both in memory and local storage, but it looks to difficult to estimate the serialized byte size of not serialized objects without serialization.
So, I added another method ```getTotalRowCount()``` which returns the total number of rows in memory and disk instead of this one. 

Using row numbers as a kind of threshold is not good because operators should estimate proper values by themselves, so I think this should be improved to return number of bytes later.